.vscode/extensions.json

```diff
@@ -1,9 +1,20 @@
 {
   "recommendations": [
-    "editorconfig.editorconfig",
-    "dbaeumer.vscode-eslint",
-    "davidanson.vscode-markdownlint",
-    "esbenp.prettier-vscode",
-    "yoavbls.pretty-ts-errors"
+    // Formating
```

Comment:
This is a .json file, should we remove comments?

Reply:
VS Code supports JSONC. Proof https://code.visualstudio.com/docs/languages/json#_json-with-comments